core/src/main/java/org/kiwix/kiwixmobile/core/utils/dialog/AlertDialogShower.kt

```diff
@@ -20,20 +20,34 @@ package org.kiwix.kiwixmobile.core.utils.dialog
 
 import android.app.Activity
 import android.app.Dialog
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.graphics.Color
+import android.text.Html
+import android.view.Gravity
+import android.widget.TextView
+import android.widget.Toast
 import androidx.appcompat.app.AlertDialog
+import androidx.core.content.ContextCompat
+import java.net.URL
 import javax.inject.Inject
 
 class AlertDialogShower @Inject constructor(private val activity: Activity) :
   DialogShower {
-  override fun show(dialog: KiwixDialog, vararg clickListeners: () -> Unit) {
-    create(dialog, *clickListeners).show()
-  }
+  private val viewSpacingLeftForLink = 0
```

Comment:
@gouri-panda Please have a look here

Reply:
Ok. Maybe we can do better at naming variable  :)

Reply:
Yeah, sure any suggestions regarding the names?

Reply:
Anything that's more easily readable are welcome!

Reply:
@gouri-panda ,
I have rename these variables.